hoomd/md/integrate.py

```diff
@@ -84,23 +84,26 @@ def methods(self, value):
 class Integrator(_DynamicIntegrator):
     R""" Enables a variety of standard integration methods.
 
-    Args: dt (float): Each time step of the simulation ```hoomd.run```
-    will advance the real time of the system forward by *dt* (in time units).
-    aniso (bool): Whether to integrate rotational degrees of freedom (bool),
-    default None (autodetect).
+    Args:
+        dt (float): Integrator time step size (in time units).
 
-    ``mode_standard`` performs a standard time step integration
-    technique to move the system forward. At each time step, all of the
-    specified forces are evaluated and used in moving the system forward to the
-    next step.
+        methods (Sequence[hoomd.md.methods._Method]): Sequence of integration
+            methods. Each integration method can be applied to only a specific
+            subset of particles, default [].
```

Comment:
Make mention that the intersection of the subsets must be null and the union must come to all the particles.

Reply:
The union does not need to be all particles.